pkg/baseline/packageIndex.json

```diff
@@ -9,8 +9,9 @@
       "BaselineVersion": "4.4.0",
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
-        "4.1.1.0": "4.4.0",
-        "4.1.2.0": "4.4.0"
+        "4.1.1.0": "4.5.0",
```

Comment:
It's primarily for validation.  Especially if your BaseLineVersion is higher than these mappings, the mappings are irrelevant based on https://github.com/dotnet/wcf/pull/1931#discussion_r117287353.